pkg_defs/ska3-flight/meta.yaml

```diff
@@ -60,4 +60,4 @@ requirements:
     - starcheck ==13.6.0
     - tables3_api ==0.1.1
     - testr ==4.3.1
-    - xija ==4.18.2
+    - xija ==4.19.0
```

Comment:
I think we decided not to update flight for now.

Reply:
And you beat me apparently pushing more commits at the instant of making the pr...

Reply:
yes, I was changing it when you were writing the comment :-)